packages/ember-simple-auth/addon/session-stores/cookie.js

```diff
@@ -287,7 +287,7 @@ export default BaseStore.extend({
 
   _renew() {
     return this.restore().then((data) => {
-      if (!isEmpty(data) && data !== {}) {
+      if (!isEmpty(data) && !(data.constructor === Object && Object.keys(data).length === 0)) {
```

Comment:
I'm not sure why `data !== {}` was added at all since [`isEmpty`](https://api.emberjs.com/ember/3.24/functions/@ember%2Futils/isEmpty) should handle the case we're trying to prevent against here anyway?

Reply:
Also unintuitively, Ember's `isEmpty` also doesn't check for empty objects: https://api.emberjs.com/ember/release/classes/@ember%2Futils/methods?anchor=isEmpty

```javascript
isEmpty({});               // false
```